Dockerfile.oraclelinux7

```diff
@@ -95,7 +95,7 @@ RUN --mount=type=cache,target=$DOWNLOADS_CACHE_DIR \
 # CMake will look for $OPENSSL_ROOT_DIR
 ENV OPENSSL_ROOT_DIR=$OPENSSL_ROOT
 RUN --mount=type=cache,target=$DOWNLOADS_CACHE_DIR \
-    source scl_source enable devtoolset-9 \
+    source scl_source enable devtoolset-10 \
```

Comment:
Search for the comments such as `gcc/g++ 9/10` in the Dockerfile. Can you update them as well?

Reply:
ok, done